### PR TITLE
create html structure and css+js components #1

### LINKED ISSUE #1: Homepage layout
-html structure
-css components (button, variables, font-family, etc)
-js components

-Header
-Hero section
-About me
-Services
-Results numbers
-Featured projects
-Feedback
-Plan prices
-Blog posts
-Logos
-Footer

### PR DESCRIPTION
**Pull request by skrendam**
-html structure
-css components
-js components
-npm install

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,8 +9,105 @@
...
+</header>
+
+<!-- Footer -->
+<header class="container">
+    <div class="row">
+        <div class="col-xs-12">
+            Footer CONTENT
+        </div>
+    </div>
+</header>
```
Comment (VytautasKaz) [inline index.html]: header --> footer

File: index.html [hunk 2]
```diff
@@ -9,8 +9,105 @@
...
+
+<!-- This is me -->
+<header class="container">
+    <div class="row">
+        <div class="col-xs-12">
+            This is me CONTENT
+        </div>
+    </div>
+</header>
```
Comment (VytautasKaz) [inline index.html]: čia ir toliau header --> section

**Review by Justas383 — DISMISSED** [review]
Looks good to me. Structure is nice and clear! Good job!

**Review by juliusvara — DISMISSED** [review]
All good

**Review by VytautasKaz — APPROVED** [review]
Good job!

**Review by Justas383 — APPROVED** [review]
Better naming of elements, good job !

**Review by justasga — APPROVED** [review]
great one!

**Pull request merged**